src/licensedcode/data/licenses/jabber.LICENSE

```diff
@@ -0,0 +1,145 @@
+Preamble
```

Comment:
This is the same as the existing josl-1.0 IMHO...
I think this should be instead a rule for the josl-1.0 and not a bew license